code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -45,6 +45,10 @@
 
 			if(!found_machine_head)								//Everyone else fails, skip the emote attempt
 				return								//Everyone else fails, skip the emote attempt
+		if("growl", "growls", "howl", "howls", "hiss", "hisses", "scream", "screams", "sneeze", "sneezes")
```

Comment:
This will break the lines below because the switch already found it's matching value in these.
Best to do this check before this switch

Reply:
Could you elaborate on that? Not too familiar with the term 'switch' in this case.

Reply:
A switch can be written out like this for instance:
```
if(case1)
  dostuff
else if(case2)
  dostuff2
else if(...)
  ....
```
Actually writing it out like that will perform worse btw due to how switches work internally. 
But due to this it will stop after the first case it finds. In your case it will find growl at line 48 and then it won't go to the lines below where it checks the cooldown